src/hb-coretext.cc

```diff
@@ -660,7 +660,7 @@ _hb_coretext_shape (hb_shape_plan_t    *shape_plan,
     scratch_size -= _consumed; \
   } while (0)
 
-  ALLOCATE_ARRAY (UniChar, pchars, buffer->len * 2, /*nothing*/);
+  ALLOCATE_ARRAY (UniChar, pchars, buffer->len * 2, ((void)0) /*nothing*/);
```

Comment:
oh, looks good, can we use nullptr here? thanks!

Reply:
yep. done.

Reply:
Oh, actually what you was making more sense now I see 😂 feel free to revert to that and I will just merge it this once bots where happy with it whether you change it what was originally or not. Thanks :)